lblod/frontend-lokaal-mandatenbeheer
# LMB-473 | Generate rows draft mandataris table

## Description

During the legislatuur preparation lokale besturen have to set an amount of mandaten per bestuursorgaan. As this can be 55 gemeenteraadsleden we want the option to generate the with default information for X amount. Add a button next to Toevoegen for generating mandaten.

## How to test

1. Go to the legislatuur page and press the generate button
2. See what happens when you want to create to much or to little mandaten
3. Generate the mandaten by pressing the button
4. The mandataris draft table should be updated with the requested rows of mandaten


## Attachments

![image](https://github.com/user-attachments/assets/f016ad2b-b713-4f1e-a7e3-bc0027d06432)

![image](https://github.com/user-attachments/assets/df2f990d-a37d-4806-9c71-290bec79df14)
